Functions.Templates/Templates/EventGridTrigger-CSharp/EventGridTriggerCSharp.csx

```diff
@@ -0,0 +1,7 @@
+#r "Microsoft.Azure.WebJobs.Extensions.EventGrid"
+using Microsoft.Azure.WebJobs.Extensions.EventGrid;
+
+public static void Run(EventGridEvent myBlob, TraceWriter log)
```

Comment:
"myBlob" is not the appropriate name for an event grid event. I would call this param eventGridEvent.